model/job.go

```diff
@@ -17,21 +17,38 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+// JobLinkProvides describes the BOSH links a job provides
+type JobLinkProvides struct {
+	Name       string
+	Type       string
+	Properties []string
+}
+
+// JobLinkConsumes describes the BOSH links a job consumes
+type JobLinkConsumes struct {
+	Name     string `json:"-"`
+	Type     string `json:"-"`
+	Optional bool   `json:"-"`
+	Role     string `json:"role"` // Name of role in which the resolved job is running
+	Job      string `json:"job"`  // Name of job that this resolves to
```

Comment:
I wonder if these 2 fields should be `RoleName` and `JobName` instead? Because right now I would expect them to be of type `*Role` and `*Job` instead of `string`, based on their name.

Reply:
e57c5c1aa5a881789004d3a7cde234396d34efe3